packages/preview-middleware-client/src/cpe/connector-service.ts

```diff
@@ -0,0 +1,33 @@
+import { ExternalAction, storageFileChanged } from '@sap-ux-private/control-property-editor-common';
+import { ActionSenderFunction } from './types';
+
+/**
+ * A Class of WorkspaceConnectorService
+ */
+export class WorkspaceConnectorService {
+    /**
+     * Initializes connector service.
+     *
+     * @param sendAction action sender function
+     */
+    public async init(sendAction: ActionSenderFunction): Promise<void> {
+        const version = sap.ui.version;
```

Comment:
We should avoid using globals. Please use `sap/ui/VersionInfo` instead here.

Reply:
Could you run `npx @ui5/linter --details` on the code? It should find such issues.

Reply:
Thanks for this command its very helpful. Probably we need an issue. there are currently 29 issues in the package preview-middleware-client. 
The one introduced with this PR is been resolved.